erica_app/erica/elster_xml/est_mapping.py

```diff
@@ -73,8 +73,6 @@
     'stmind_pflegekosten_anspruch': 'E0161405',
     'stmind_beh_aufw_summe': 'E0161504',
     'stmind_beh_aufw_anspruch': 'E0161505',
-    'stmind_beh_kfz_summe': 'E0161604',
-    'stmind_beh_kfz_anspruch': 'E0161605',
```

Comment:
I think you'll still want to adapt the acceptance tests?

Reply:
Great catch!